editor/project_manager/project_list.cpp

```diff
@@ -242,6 +265,8 @@ ProjectListItemControl::ProjectListItemControl() {
 	favorite_button->set_mouse_filter(MOUSE_FILTER_PASS);
 	favorite_box->add_child(favorite_button);
 	favorite_button->connect(SceneStringName(pressed), callable_mp(this, &ProjectListItemControl::_favorite_button_pressed));
+	favorite_button->connect(SceneStringName(focus_entered), callable_mp((CanvasItem *)favorite_button, &CanvasItem::set_self_modulate).bind(EDITOR_GET("interface/theme/accent_color")));
```

Comment:
This won't work if you change accent color, though not sure if it's possible in Project Manager.

Reply:
Now, use icon to indicate favorite or not, and use `self_modulate` to indicate focus or not. So there will be no conflict.

https://github.com/user-attachments/assets/5b26c1cb-3577-4ba8-a9b7-2dd8ca01077f

Reply:
What I meant is that you bind a constant color. If you somehow change accent color, the button will still use the old one for `self_modulate`. I think this can happen when you have custom accent color and change color scheme via Settings.

Reply:
Yeah, you are right.